error-handling.js

```diff
@@ -0,0 +1,21 @@
+exports.handlePSQL400s =(err, request, response, next) => {
```

Comment:
Very nice error handling functions. I would maybe make the function name more generic, just in case you end up using it to handle status codes other than 400